pkg/istio-agent/sds-agent.go

```diff
@@ -88,7 +88,7 @@ var (
 )
 
 const (
-	// name of authentication provider.
+	// name of authentication provider. Specify empty if proxy uses file mounted certs and provisioning is not needed.
```

Comment:
CA_PROVIDER is for the provider of the CA. Setting CA_PROVIDER as empty to express the meaning that proxy uses file-mounted certs seems to be a hack and makes the semantic of CA_PROVIDER confusing.
Instead of setting CA_PROVIDER as empty, I suggest two alternatives: 
1. Detecting whether proxy uses file mounted certs or not.
2. Use a new variable to represent whether proxy uses file mounted certs or not.